spellbook/models/magiceden/solana/magiceden_solana_events.sql

```diff
@@ -1,10 +1,11 @@
 {{ config(
-        alias ='events',
-        materialized ='incremental',
-        file_format ='delta',
-        incremental_strategy='merge',
-        unique_key='unique_trade_id'
-        )
+    alias = 'events',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'unique_trade_id']
```

Comment:
this is very useful, instead of making a new key. nice.

Reply:
could consider this to remove unique_trade_id potentially in the future 🤔